fitnessllm_dataplatform/stream/strava/etl_utils.py

```
@@ -0,0 +1,243 @@
+"""ETL Utilities for Strava Data Source."""
+import itertools
+import json
+import multiprocessing as mp
+from dataclasses import asdict
+from datetime import datetime
+from enum import Enum, EnumType
```

Ruff (0.8.2)

`enum.Enum` imported but unused

Remove unused import: `enum.Enum`

(F401)